core/src/main/java/org/apache/druid/metadata/MetadataStorageActionHandler.java

```diff
@@ -130,6 +130,13 @@ void insert(
    */
   void removeLock(long lockId);
 
+  /**
+   * Remove the tasks created order than the given createdTime.
```

Comment:
nit: change "order" or "older". Or may be better way is "Remove the tasks with timestamp older than given timestamp